### PR TITLE
feat(input): inline form field

### LINKED ISSUE #482: Streamlined input control
**Describe the problem this feature request is solving**
I and others are needing an input control which gives a similar functionality to the current hc-form-field and hc-input, but we need it to be more streamlined(label to the left with little margin on top/bottom)

**Describe the solution you'd like**
It doesn't make sense to completely rewrite a new component for this feature

**Describe alternatives you've considered**
The user at the moment is required to put their own label to the left and override the buttons top/bottom margin, which is a little awkward at the moment.

** If any, which components will have to change, and how **
form-field should take an input such as `streamLined:boolean` this will just require some modification to the scss but the functionality of the form-field and input will remain unchanged

### PR DESCRIPTION
**Pull request by andrew-frueh**
Adds new flag to hc-form-field which allows for inline display - which will apply to select boxes also as soon as select is integrated with hc-form-field.  Consolidated the "disabled" input example into the prefix/suffix example.

closes #482

### CODE REVIEW

File: projects/cashmere/src/lib/input/hc-form-field.component.ts [hunk 1]
```diff
@@ -29,6 +32,16 @@ export class HcFormFieldComponent implements AfterContentInit {
...
     }
 
+    /** Whether the form elements should be stacked (default), or inline */
+    @Input()
+    get inline(): boolean {
+        return this._inline;
+    }
+
+    set inline(isInline) {
+        this._inline = parseBooleanAttribute(isInline);
```
[the remark this answers is not in the document]
Comment (andrew-frueh) [reply projects/cashmere/src/lib/input/hc-form-field.component.ts]: That is an excellent question @jonest and it turns out the answer is no.  That function has been in utils.ts forever and you're the first person to bring that up, so thank you!  I've logged an issue and will submit a separate PR with a fix.

**Comment by jonest** [comment]
Checking these browsers:
- [x] Firefox
- [ ] IE
- [x] Chrome
- [ ] Edge

IE I can't get to load:
image

Edge I see this:
image

**Comment by andrew-frueh** [comment]
Unfortunately, the IE and Edge compatibility is bigger than this PR 😞.  Issues are on the list for getting both of those resolved: #200 and #491

**Comment by jonest** [comment]
> Unfortunately, the IE and Edge compatibility is bigger than this PR 😞. Issues are on the list for getting both of those resolved: #200 and #491

Bummer 😢 Looks good aside from that then!

**Pull request merged**